frontend/components/forms/fields/Dropdown/_styles.scss

```
@@ -8,6 +8,9 @@
     &-label {
       line-height: 38px;
       font-size: $x-small;
+      // Override react-select's hardcoded `.Select-value { color: #aaa; }` so
```

Stylelint (17.7.0)

[error] Expected empty line before comment (scss/double-slash-comment-empty-line-before)

(scss/double-slash-comment-empty-line-before)